easybuild/scripts/rpath_args.py

```diff
@@ -106,13 +112,14 @@
 # add -rpath flags in front
 cmd_args = cmd_args_rpath + cmd_args
 
+cmd_args_rpath = []
+if rpath_include:
+    for path in rpath_include:
+        cmd_args_rpath.append(flag_prefix + '-rpath=%s' % path)
```

Comment:
This can be collapsed to a single line, since we're sure `rpath_include` is a list (possibly empty, but that's fine):

```python
cmd_args_rpath = [flag_prefix + '-rpath=%s' % inc for inc in rpath_include]